### DISCUSSION TITLE
Can I file an issue to request for a new formula to be added?

### DISCUSSION THREAD

**Discussion by jasjuang** [Homebrew/discussions#2049] [Getting started]
I am specifically interested in seeing Alembic: https://github.com/alembic/alembic to be added into the brew ecosystem. However, when I search the issues in homebrew-core, I cannot find any new formula requests, but in contrast there are constant new port requests in the issues of vcpkg, and the maintainers address them. Since I am fairly new to the brew ecosystem, can someone let me know where is the right place for me to submit new formula requests?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Reply by jasjuang** [reply]
If I understand correctly, does this imply I cannot request the maintainers to add them?

**Reply by SMillerDev** [reply]
Well, you can make a pull request to request that they merge the formula you wrote. You can't request the maintainers to do all the work for you, no.

**Reply by iMichka** [reply]
There are also platforms out there were you can pay developers to do it for you. It’s just that the maintainers here have only time to maintain the current formulae, but don’t have much time for other things like adding your favorite package.

**Comment by alebcay** [comment]
`alembic` is available already from `brewsci/science`: https://github.com/brewsci/homebrew-science/blob/master/Formula/alembic.rb

You can install this with `brew install brewsci/science/alembic`. The version is a bit out of date but it looks like the package would qualify to be moved to the main core tap if there's interest in that.

**Reply by alebcay** [reply]
Considering that `brewsci/science` is unmaintained, opened PR for migration to core in https://github.com/Homebrew/homebrew-core/pull/84228.
